paimon-core/src/main/java/org/apache/paimon/operation/FileStoreWrite.java

```diff
@@ -125,4 +126,6 @@ List<CommitMessage> prepareCommit(boolean waitCompaction, long commitIdentifier)
 
     /** With metrics to measure compaction. */
     FileStoreWrite<T> withMetricRegistry(MetricRegistry metricRegistry);
+
+    WriterMetrics getWriterMetrics();
```

Comment:
Why do you add this method in the interface when it is only used internally by `AbstractFileStoreWrite` and its subclasses?

Reply:
done